samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor

```diff
@@ -113,33 +113,49 @@
         @foreach (var node in PagedNodeList.Results)
         {
         <tr>
-            <td>@node.NodeName</td>
-            <td>@node.Id</td>
+            <td class="hover-text width-medium">
+                <div>
+                    @node.NodeName
+                </div>
+            </td>
+            <td class="hover-text width-large1">
+                <div>
+                    @node.Id
+                </div>
+            </td>
             @if (node.Children && (node.NodeClass != NodeClass.Method))
             {
-                <td>
-                    <a href="javascript: void(0)" @onclick="@(() => GetTreeAsync(node.Id, node.ParentIdList))">
-                        @node.NodeClass
-                    </a>
+                <td class="hover-text width-small">
+                    <div>
+                        <a href="javascript: void(0)" @onclick="@(() => GetTreeAsync(node.Id, node.ParentIdList))">
+                            @node.NodeClass
+                        </a>
+                    </div>
                 </td>
             }
             else if (node.NodeClass == NodeClass.Method)
             {
-                <td>
-                    <a href="javascript: void(0)" @onclick="@(() => OpenDrawer(node, drawer.Action))">
-                        @node.NodeClass
-                    </a>
+                <td class="hover-text width-small">
+                    <div>
+                        <a href="javascript: void(0)" @onclick="@(() => OpenDrawer(node, drawer.Action))">
+                            @node.NodeClass
+                        </a>
+                    </div>
                 </td>
             }
             else
             {
-                <td>@node.NodeClass</td>
+                <td class="hover-text width-small">
+                    <div>
+                        @node.NodeClass
+                    </div>
+                </td>
             }
             @if (node.NodeClass == NodeClass.Variable && (((node.AccessLevel & NodeAccessLevel.CurrentRead) == NodeAccessLevel.CurrentRead) ||
                ((node.AccessLevel & NodeAccessLevel.CurrentWrite) == NodeAccessLevel.CurrentWrite)))
             {
-                <td class="value-wide value-scroll">
-                    <div>
+                <td class="hover-text width-large1">
+                    <div class="value-scroll">
```

Comment:
Here we get extremely long lines 

![image](https://user-images.githubusercontent.com/11295647/83062481-01c13d00-a05f-11ea-8c2c-690d3ea1a1ed.png)

Reply:
fixed